libafl_qemu/Cargo.toml

```diff
@@ -87,11 +87,13 @@ enum-map = "2.7"
 serde_yaml = { version = "0.8", optional = true } # For parsing the injections yaml file
 toml = { version = "0.4.2", optional = true } # For parsing the injections toml file
 pyo3 = { version = "0.18", optional = true }
+rustversion = "1.0"
 # Document all features of this crate (for `cargo doc`)
 document-features = { version = "0.2", optional = true }
 
 [build-dependencies]
 pyo3-build-config = { version = "0.18", optional = true }
+rustversion = "1.0"
```

Comment:
Isn't listing it in normal deps enough? (or only build deps)

Reply:
I couldn't use it in one of them for some reason.
The build dependency is necessary to load the `used_with_arg` feature only in nightly.
The normal dependency is used to enable the feature conditionally for nightly only.

For some reason, using the `nightly` feature out of `lib.rs` did not work on my side.
I'll give it a try again, maybe I missed something.

Reply:
I just checked and now it seems to work as intended for some reason.